packages/fxa-content-server/app/scripts/views/mixins/modal-panel-mixin.js

```diff
@@ -45,6 +45,44 @@ module.exports = {
 
     this._boundBlockerClick = this.onBlockerClick.bind(this);
     $('.blocker').on('click', this._boundBlockerClick);
+
+    /**
+     * Trap keyboard focus when modal opens.
+     */
+    const modal = this.$el[0];
+
+    modal.addEventListener('keydown', tabClick);
```

Comment:
Can you add an `events` property to the mixin and add the handler there? That will ensure event housekeeping is taken care of, e.g., listeners are removed when the view is destroyed. Doing so might require ignoring keydown events while the panel is closed.

Reply:
Alright, let me do that.